pyretis/meta.yaml

```diff
@@ -0,0 +1,60 @@
+#{% set name = "pyretis" %}
+#{% set version = "0.9.2" %}
+#{% set sha256 = "f738210a323dd8f84f98f33f0836dfdcc04f3fa8559a4e847f8eb311f147c7cc" %}
+
+package:
+   name: pyretis
+   version: 0.9.2
+#  name: {{ name|lower }}
+#  version: {{ version }}
+
+source:
```

Comment:
Is it possible to use the source code from pypi.io (0.9.2.dev2)? See:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos